desktop/ui/src/main/java/org/datacleaner/windows/DataHubDatastoreDialog.java

```diff
@@ -264,6 +286,11 @@ protected boolean validateForm() {
             return false;
         }
 
+        if (_tenantName == null) {
+            setStatusError("Please test the connection first");
+            return false;
```

Comment:
I think this then prevents you from saving a datastore without testing it? Isn't that a bit annoying? What if you're simply editing an existing datastore or maybe you temporarily don't have access to the datahub? Testing the datastore can IMO be postponed till the moment it is actually connecting. I guess also username/passwords may change and all that, so verifying it at this time is only giving a false sense of verification.